scrum-of-scrums/readme.md

```diff
@@ -1,23 +1,33 @@
 # Scrum of Scrums
 
-] This content is under construction. For now, for those who are unfamiliar: [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)
+Scrum or scrums is a technique used to scale Scrum to a larger group. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.
 
 ## Goals
 
-What are the intended outcomes of the ceremonies?
+The goal of the scrum of scrums ceremony is to give sub teams the agility they need while not loosing visibility and coordination.
+
+The scrum of scrums ceremony happens every day and can be seen as a regular [stand-up](../stand-ups/readme.md):
+
+    - What was done the day before by the sub team
+    - What will be done today by the sub team 
+    - What are blockers or other issues for the sub team
+
+The outcome of the meeting will result in a list of impediments related to coordination of the whole project. Solutions could be: agreeing on interfaces between teams, discussing architecture changes, evolving responsility boundaries, etc.
+
+This list of impediments is usually managed in a separate [backlog](../backlog-management/readme.md) but does not have to.
 
 ## Participation
 
-What members of the team should participate?
+The common guideline is to have on average one person per sub team to participate in the scrum of scrums. Ideally, the scrum master of each sub team would represent them in this ceremony. In some instances, the representative for the day is selected at the end of each sub team daily [stand-up](../stand-ups/readme.md) and could change every day. In practice, having a fixed representative tends to be more efficient in the long term.
 
 ## Impact
 
-What positive impact has been observed as a result of this practice?

```

Comment:
List also the drawbacks. For example: some team members will have additional meetings to coordinate and participate. All team members need to be synched later to guarantee a good flow of information etc.